### DISCUSSION TITLE
Can we assign just a project and its reporting to an administrator

### DISCUSSION THREAD

**Discussion by bhushan619** [kimai/kimai#3465] [Questions & Answers]
I have multiple projects and multiple admins for the respective projects.. I just want an admin to view the project details team an reporting associated with his respective project.. can this be done? As of now admin have rights to see all the projects and reporting..

**Comment by kevinpapst** [comment]
Teams limit access to content: 
https://www.kimai.org/documentation/teams.html

Giving admin permissions should be avoided, use Teamlead roles and assign further permissions: 
https://www.kimai.org/documentation/permissions.html

**Reply by bhushan619** [reply]
Thank you Kevin i studied it and changed the role administrator to team lead, now the team lead can view the report with the project assigned to him and cannot see other project reports.. but in reporting he cannot view the 'users' tab.. what can I do to make it visible?

**Reply by kevinpapst** [reply]
Check permissions `view_other_timesheet` and `view_other_reporting` for the teamlead role

**Reply by kevinpapst** [reply]
And the team members probably have to be in the same team as the teamlead
